ambry-clustermap/src/main/java/com/github/ambry/clustermap/InstanceConfigToDataNodeConfigAdapter.java

```diff
@@ -75,6 +75,10 @@ public DataNodeConfig get(String instanceName) {
     return instanceConfig != null ? converter.convert(instanceConfig) : null;
   }
 
+  @Override
+  public void close() {
+  }
```

Comment:
what's this close method for? Does the helixManager needs to be disconnected here?

Reply:
This close method is just here since I added close() to the interface, but this specific impl shouldn't have to do anything. Unlike PropertyStore...Adapter, this one doesn't own the `HelixManager`'s lifecycle, since it was passed into the constructor. HelixClusterManager will disconnect the `HelixManager`.